docs/guides/transactions.md

```diff
@@ -6,6 +6,11 @@ A transaction is basically a context manager that collects all file uploads, def
 They are an "all or nothing" proposition: If an error occurs during the transaction, none of the queued files are uploaded.
 For more information on fsspec transactions, see the official [documentation](https://filesystem-spec.readthedocs.io/en/latest/features.html#transactions).
 
+!!! info
+    The transactions in `lakeFS-spec` are different from the transactions in the [lakeFS-SDK](https://docs.lakefs.io/integrations/python.html#transactions), which were added in `v0.2.0`. 
+    - `lakeFS-SDK` transactions create an ephemeral branch, perform the operations in the context block on this branch and, upon exiting the context manager, merge it back into the source branch. 
```

Comment:
```suggestion
    - High-level `lakefs` SDK transactions create an ephemeral branch, perform the operations in the context block on that ephemeral branch and, upon exiting the context, merge it back into the source branch. 
```

Reply:
I would leave out the backticks (as mentioned above).